etna/articles/migrations/0061_alter_articlepage_mark_new_on_next_publish.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.0.8 on 2023-02-24 16:20
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("articles", "0060_articlepage_mark_new_on_next_publish_and_more"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="articlepage",
+            name="mark_new_on_next_publish",
+            field=models.BooleanField(
+                default=True,
+                help_text="Mark this as true before publishing, if you want to display the 'new label' for 3 weeks",
+                verbose_name="display a new label on the card of this page",
```

Comment:
Maybe a simple "Mark this page as 'new' when published" would make for a clearer label? 

The help text feels a bit overly wordy too. Perhaps we should just confirm where the indicator appears there, and leave the number of days up to whatever has been configured?